### DISCUSSION TITLE
Vietnamese Language Leader

### DISCUSSION THREAD

**Discussion by TaQuangTien** [OHF-Voice/intents#412] [Language Leaders]
I'm willing to be Vietnamese Dev_Voice leader.  taquangtien92@gmail.com
I'm on Discord channel now.

**Comment by synesthesiam** [comment]
Thanks @TaQuangTien! I'll add you to the list.

**Comment by synesthesiam** [comment]
Empty Vietnamese templates are in the repository :+1: 
Please start with `_common.yaml` and the `homeassistant*.yaml` files in `sentences/vi`

**Comment by dinhchinh82** [comment]
Hello @synesthesiam 
I'm interested in making smart speaker for Vietnamese. With some experience on Vietnamese NLP since 2018, I'm very excited to contribute on Rhasspy project to make it well support on Vietnamese language.

I've just submited  the first [PR](https://github.com/home-assistant/intents/pull/312) for Vietnamese. I'm willing to be the language leader of Vietnamese.
Thank you.

**Reply by synesthesiam** [reply]
Thanks! Let me check out the PR.

I've been able to gather more Vietnamese data over the years, so I should be able to start training speech to text and text to speech models soon :slightly_smiling_face:

**Reply by dinhchinh82** [reply]
That's great.

I'm willing to support you to bring Vietnamese to the more smart speakers.

> Thanks! Let me check out the PR.
> 
> I've been able to gather more Vietnamese data over the years, so I should be able to start training speech to text and text to speech models soon 🙂

**Comment by synesthesiam** [comment]
@dinhchinh82 let's get [this PR](https://github.com/home-assistant/intents/pull/312) merged and then I'll add you as language leader :+1:

**Reply by dinhchinh82** [reply]
> @dinhchinh82 let's get [this PR](https://github.com/home-assistant/intents/pull/312) merged and then I'll add you as language leader 👍

@synesthesiam I just updated new commit. Please review that PR again.

**Reply by synesthesiam** [reply]
Thanks @dinhchinh82! Adding you as a language leader.

**Comment by synesthesiam** [comment]
@dinhchinh82 Would you be able to translate the new responses files into Vietnamese?

**Reply by dinhchinh82** [reply]
@synesthesiam yes, I will update new response files soon.

**Comment by synesthesiam** [comment]
@dinhchinh82 Are you on Discord?

**Reply by dinhchinh82** [reply]
@synesthesiam I just sent you message via Discord. My Discord is chinhhut. Thank you.
